Tests/configure_and_test_integration_instances.py

```diff
@@ -1296,13 +1297,79 @@ def set_marketplace_url(servers, branch_name, ci_build_number):
     sleep(60)
 
 
-def main():
-    install_logging('Install Content And Configure Integrations On Server.log')
-    build = Build(options_handler())
+def test_integrations_post_update(build: Build, new_module_instances: list, old_module_instances: list) -> tuple:
+    """
+    Runs 'test-module on all integrations for post-update check
+    Args:
+        build: A build object
+        new_module_instances: A list containing new integrations instances to run test-module on
+        old_module_instances: A list containing old (existing) integrations instances to run test-module on
+
+    Returns:
+        * A list of integration names that have failed the 'test-module' execution post update
+        * A list of integration names that have succeeded the 'test-module' execution post update
+    """
+    old_module_instances.extend(new_module_instances)
+    successful_tests_post, failed_tests_post = instance_testing(build, old_module_instances, pre_update=False)
+    return failed_tests_post, successful_tests_post
```

Comment:
try to be consistent, if `success` is returned before ` failed` in one function, keep that order through the entire flow.